### PR TITLE
Documentation for the Apache Ignite Handler

### PR DESCRIPTION
**Pull request by MinuraPunchihewa**
Added the documentation for the Apache Ignite handler.

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
Related to: https://github.com/mindsdb/mindsdb/issues/6770

**Comment by martyna-mindsdb** [comment]
@MinuraPunchihewa 
Please see the review comments above and push changes to this PR. Thanks.

**Comment by martyna-mindsdb** [comment]
@MinuraPunchihewa 
Now that the Apache Ignite handler works fine, let's finish up this PR.

File: docs/data-integrations/apache-ignite.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+[Apache Ignite](https://ignite.apache.org/docs/latest/) is a distributed database for high-performance computing with in-memory speed.
+
+## Implementation
+
+This handler is implemented using the `pyignite` library, the Apache Ignite thin (binary protocol) client for Python.
+
+The required arguments to establish a connection are as follows:
+
+* `host` is the host name or IP address of the Apache Ignite cluster's node.
+* `port` is the TCP/IP port of the Apache Ignite cluster's node. Must be an integer.
```
Comment (martyna-mindsdb) [inline docs/data-integrations/apache-ignite.mdx:17]: Please list all parameters here, including host, port, username, password, and schema.

**Comment by martyna-mindsdb** [comment]
Thanks @MinuraPunchihewa 
Let's list all connection parameters (as in the review comment above).

File: docs/data-integrations/apache-ignite.mdx (hunk 1 above)
Comment (MinuraPunchihewa) [reply docs/data-integrations/apache-ignite.mdx:17]: Apologies for missing this @martyna-mindsdb. It's done now.

Comment (martyna-mindsdb) [reply docs/data-integrations/apache-ignite.mdx:17]: Thank you @MinuraPunchihewa

**Pull request merged**